string cell
---
geometry_msgs/TransformStamped p1
geometry_msgs/TransformStamped p2
